### ISSUE TITLE
Fine-tune a pre-sparsified YOLOv5l6 1280 model

### ISSUE THREAD

**Issue by omaralnasier** [neuralmagic/sparseml#1622]
Reciepe for Yolov5l6 is not currently available for reproducing, so how I can use transfer laearning on my custom data? I use this CLI:

sparseml.yolov5.train \
  --cfg yolov5l.yaml \
  --weights zoo:cv/detection/yolov5-l6/pytorch/ultralytics/coco/pruned75_quant-none?recipe_type=transfer_learn \
  --recipe zoo:cv/detection/yolov5-l6/pytorch/ultralytics/coco/pruned75_quant-none?recipe_type=transfer_learn \
  --data my-custom-data.yaml \
  --patience 0 \
  --hyp hyps/hyp.finetune.yaml

I used it before on a yolov5s model, with the corresponding changes to the recipe, weights ,and the model arch it works fine.
However it produces a yaml loading error when I tried it for my large model (yolov5l6 1280), after doing some checking I see that you say that the recipe for reproducing the results for this model(you mean training) it is not currently available.

https://sparsezoo.neuralmagic.com/models/yolov5-l6-coco-pruned75_quantized?comparison=yolov5-l6-coco-base&tab=1

My question is : how I can fine-tuning a pre-sparsified YOLOv5l6model ?

**Comment by anmarques** [comment]
Hi @omaralnasier. I have pushed a transfer learning recipe to YOLOv5l6 pruned75_quant. You can use the command you mentioned above to do the transfer learning (please use --cfg yolov5l6.yaml).

Also, please feel free to experiment w/ hyper-parameters for your specific use case. For instance, you can change the number of epochs and quantization learning rate by using:

`--recipe_args '{"num_epochs":30,"quantization_lr":5e-5}'
`
